client/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Product Description</title>
+</head>
+<body>
+  <div id="description"></div>
```

Comment:
This may cause problems on the stitching, I would recommend changing it to like tbay-products-description or the like.